.github/workflows/codespell.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: Codespell
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  codespell:
+    name: Check for spelling errors
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Codespell
+        uses: codespell-project/actions-codespell@v1
```

Comment:
Can you change this to a job in `contrib.yml`?

Reply:
done in 5451bcdaaf4c31f6391abad04891e8699b33282f